src/bundle/Resources/views/content/tabs/relations/table.html.twig

```diff
@@ -0,0 +1,40 @@
+{% trans_default_domain "locationview" %}
+
+{% if relationList %}
+    <table class="ez-table-data">
+        <thead>
+            <tr>
+                <th>{{ 'locationview.relations.name'|trans()|desc('Name') }}</th>
+                <th>{{ 'locationview.relations.content.type'|trans()|desc('Content type') }}</th>
+                <th>{{ 'locationview.relations.relation.type'|trans()|desc('Relation type') }}</th>
+            </tr>
+        </thead>
+        <tbody>
+        {% for relation in relationList %}
+            <tr>
+                <td><a href="{{ path(relation.destinationLocation) }}">{{ relation.destinationContentInfo.name }}</a></td>
+                <td>{{ relation.destinationContentType }}</td>
+                <td>
+                    {% if relation.type == constant('eZ\\Publish\\API\\Repository\\Values\\Content\\Relation::COMMON') %}
+                        {{ 'locationview.relations.type.content_level_relation'|trans()|desc('Content level relation') }}
+                    {% endif %}
+                    {% if relation.type == constant('eZ\\Publish\\API\\Repository\\Values\\Content\\Relation::EMBED') %}
+                        {{ 'locationview.relations.type.embed'|trans()|desc('Embed') }}
+                    {% endif %}
+                    {% if relation.type == constant('eZ\\Publish\\API\\Repository\\Values\\Content\\Relation::LINK') %}
+                        {{ 'locationview.relations.type.link'|trans()|desc('Link') }}
+                    {% endif %}
+                    {% if relation.type == constant('eZ\\Publish\\API\\Repository\\Values\\Content\\Relation::FIELD') %}
+                        {{ 'locationview.relations.type.attribute'|trans()|desc('Attribute') }}
```

Comment:
As mentioned in the jira issue (Label change):
> When the relation is field specific, then the value in the relation type column will read 'Field (%name_of_Field%)' and NOT 'Attribute (%name_of_Field%)'

So I think it should be 
`{{ 'locationview.relations.type.attribute'|trans()|desc('Field') }}`